recipes/pycraf/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "pycraf" %}
+{% set version = "0.25.7" %}
+{% set sha256 = "f17bf89af7f9a708a6aa1b79ad24a48a09754dbe39a1edf03b27f78d7bb70820" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  skip: True  # [py27]
+
+requirements:
+  build:
+    - python  # [win]
+    - vc 14   # [win and py35]
+    - vc 14   # [win and py36]
+    - {{ compiler('cxx') }}
+  host:
+    - openmp 4.0.*  # [osx]
+    - python
+    - pip
+    - setuptools
+    - astropy >=2.1
+    - cython >=0.27
+    - numpy >=1.11
+    - scipy >=1.0
+    - pytest
```

Comment:
It is odd that `pytest` is need to build and run. This is either a packaging problem upstream or you probably don't need it at all.